src/main/java/org/junitpioneer/jupiter/report/PioneerReport.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter.report;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+
+/**
+ * Root element of the Pioneer report.
+ */
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "", propOrder = { "properties", "testContainer", "testCase" })
+@XmlRootElement(name = "pioneerReport")
+public class PioneerReport {
```

Comment:
This class could just extend `TestContainer` and have an empty body - in which case you'd need to remove `propOrder`. This would solve the code duplication issue with Sonar.